sentry-core/src/scope/real.rs

```diff
@@ -341,6 +341,14 @@ impl Scope {
         self.span.as_ref().clone()
     }
 
+    /// Returns the currently active transaction.
+    pub fn get_transaction(&self) -> Option<TransactionOrSpan> {
```

Comment:
```suggestion
    pub fn get_transaction(&self) -> Option<Transaction> {
```

`get_transaction` should always return a transaction, so we should enforce this through the type system.